File: KPP_lab_1_2/KPP_lab_1_2/Characteristic.cs
namespace KPP_lab_1_2
{
    [Serializable]
    internal class Characteristic
    {
        private string name;
        private int value;

        public Characteristic(string name, int value)
        {
            this.name = name;
            this.value = value;
        }

        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        public int Value
        {
            get { return this.value; }
            set { this.value = value; }
        }
    }
}

File: KPP1_5/src/main/java/com/university/entities/Characteristic.java
package com.university.entities;

import java.io.Serial;
import java.io.Serializable;

public class Characteristic implements Serializable {
    private String name;
    private int value;

    @Serial
    private static final long serialVersionUID = 41242L;

    public Characteristic(String name, int value) {
        this.name = name;
        this.value = value;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getValue() {
        return value;
    }

    public void setValue(int value) {
        this.value = value;
    }

}
